docs/advanced-usage.rst

```diff
@@ -117,8 +117,8 @@ You can use :class:`~poolmanager.ProxyManager` to tunnel requests through an
 HTTP proxy::
 
     >>> import urllib3
-    >>> proxy = urllib3.ProxyManager('http://localhost:3128/')
-    >>> proxy.request('GET', 'http://google.com/')
+    >>> http = urllib3.ProxyManager('http://localhost:3128/')
+    >>> http.request('GET', 'http://google.com/')
```

Comment:
@sethmlarson I think I liked the `proxy` name which makes it clear that the request is going through a proxy, but don't have any strong opinion. What's the rationale for using `http` here?

Reply:
I'm ++ on using `proxy` here, I'll update the rules.